deploy/terraform-datacommons-website/modules/vpc/variables.tf

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "create_vpc" {
+  type        = bool
+  description = "Specifying true will create a new VPC network."
+  default     = true
+}
+
+variable "project_id" {
+  type        = string
+  description = "GCP project id where the VPC will be created."
+}
+
+variable "network_name" {
+  type        = string
+  description = "The name of the network being created, or the name of a existing VPC network."
+  default     = "data-commons-network"
+}
+
+variable "subnet_name" {
+  type        = string
+  description = "The name of the subnet being created to host the cluster in."
+  default     = "data-commons-subnet"
```

Comment:
Is it able to change the current deployment subnet_name to this? Or would there be discrepancy between the config and existing deployments?

Reply:
Terraform doesn't seem to do a good job with changing existing deployment. It will likely try to delete and recreate the VPC, which means deleting and recreating the cluster that depends on it, and there is a lot of risk.

For changing the params of existing infra, I think it might be easier to manually edit it through UI.